.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "Feature Request"
+about: Check instructions for submitting your idea on the mailing list first.
+
+---
+
+## Feature
+
+If you're looking to request a new feature or change in functionality, including
+adding or changing the meaning of arguments to an existing function, please
+post your idea on the [numpy-discussion mailing list]
+(https://mail.python.org/mailman/listinfo/numpy-discussion) to explain your
+reasoning in addition to opening an issue or pull request. You can also check
```

Comment:
My fear is we will end up with users submitting issues that include this template, which isn't useful. Would wrapping in `<!-- -->` make sense here?

Reply:
Good catch. Done!